src/utils/date-utils.js

```diff
@@ -4,9 +4,30 @@ const _dayJs = dayjs.extend(relativeTime);
 
 /**
  * Take a timestamp and return a meaningful length of time, e.g. 5 months ago
- * @param {ISO 8601 string} timestamp The timestamp to be converted
+ * Kedro uses this format VERSION_FORMAT = "%Y-%m-%dT%H.%M.%S.%fZ"
+ * So we need to do some string manipulation to get it to this formatted
+ * version: 2021-11-08T18:31:01.171Z
+ * @param {string} timestamp The timestamp to be converted
  * @returns A human-readable from-now date
  */
 export const toHumanReadableTime = (timestamp) => {
-  return _dayJs(timestamp).fromNow();
+  const splitTimestamp = timestamp.split('.');
+  let formattedTimestamp = '';
+
+  let i = 0;
+  const length = splitTimestamp.length;
+
+  while (i < length) {
+    if (i < 2) {
+      formattedTimestamp += `${splitTimestamp[i]}:`;
+    } else if (i === 2) {
+      formattedTimestamp += `${splitTimestamp[i]}.`;
+    } else {
+      formattedTimestamp += `${splitTimestamp[i]}`;
+    }
+
+    i++;
+  }
```

Comment:
can we simplify this with the `.replace` method with Regex ( obviously we need to find a Regex pattern than matches and replace the first 2 occurrences of the `.` instances in the string, but from what I recall that is doable with Regex, though this might require more digging in with Regex patterns.)

Reply:
I updated it once more after that push. [See here](https://github.com/quantumblacklabs/kedro-viz/commit/77f04d68f38badfff62e0bb2a889301228cd40c9). It's a one-liner again.

Reply:
the new changes looks great - thank you for putting that in 😄